src/v2/ion/responseTransformer.js

```diff
@@ -120,6 +120,26 @@ const injectIdPConfigButtonToRemediation = (settings, idxResp) => {
   return idxResp;
 };
 
+/**
+ * Whenever there is only one remediation form and it is `redirect-idp`,
+ * widget has to automatically redirect following link from `redirect-idp` form.
+ *
+ * The idea now is to reuse `success-redirect` thus converts `redirect-idp` to `success-redirect` form.
+ */
+const convertRedirectIdPToSuccessRedirectIffOneIdp = (result) => {
+  if (Array.isArray(result.remediations)) {
+    const redirectIdpRemediations = result.remediations.filter(idp => idp.name === RemediationForms.REDIRECT_IDP);
+    if (redirectIdpRemediations.length === 1 && result.remediations.length === 1) {
+      const successRedirect = {
+        name: RemediationForms.SUCCESS_REDIRECT,
```

Comment:
Although this seems ok to, I am wondering what the ideal approach here would be. If there is only one idp I am guessing that we are not going to even render any idp buttons, which means we are not making use of the idp object in the API. Should the API send back a success redirect form in this case with the href as the idp redirect url, because thats essentially what we are doing here on the client side.

Reply:
the feedback from backend is backend believe it's ux concern about auto redirect. 
other people could implement as display idp button regardless how many  redirect-idp remediation.

My thinking if reuse `success-redirect` doesn't quite make sense, we can coin a new remediation like `auto-redirect-idp` which only meant for widget.

Reply:
I am not against falling back to `success-redirect`.  Re-using it seems better than creating `auto-redirect-idp`.